test/componentWithName.jsx

```diff
@@ -217,6 +217,48 @@ describe('componentWithName', () => {
     ));
   });
 
+  describe('when a regex value is provided instead of a string', () => {
+    it('passes with an SFC', () => assertPasses(
+      componentWithName(new RegExp(/FC$/)),
```

Comment:
Do any of these test cases need `new RegExp`? They seem like they could all just be literals.